include/crm/common/cmdline.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU General Public License version 2
```

Comment:
probably LGPL

Reply:
Ah, I grabbed `cib_secrets.h` as a template to use when starting this new file.  I'll change `cmdline_internal.h`, but maybe it also needs to be changed?  I'm unfamiliar with the licensing on pacemaker stuff.